frontend/packages/console-app/src/components/modals/clone/clone-pvc-modal.tsx

```diff
@@ -118,15 +150,24 @@ const ClonePVCModal = withHandlePromise((props: ClonePVCModalProps) => {
               aria-label="Clone Pvc"
             />
           </FormGroup>
-          <FormGroup label="Size" isRequired fieldId="clone-pvc-modal__size">
+          <FormGroup
+            label="Size"
+            isRequired
+            fieldId="clone-pvc-modal__size"
+            className="co-clone-pvc-modal__ocs-size"
+          >
             <RequestSizeInput
               name="requestSize"
               onChange={requestedSizeInputChange}
               defaultRequestSizeUnit={requestedUnit}
               defaultRequestSizeValue={requestedSize}
-              dropdownUnits={dropdownUnits}
+              dropdownUnits={filterUnits()}
+              isInputDisabled={isCephProvisioner(scResource?.provisioner)}
               required
             />
+            <span className="help-block co-clone-pvc-modal__helpblock">
+              {!validSize ? 'Size should be equal or higher than default size' : <br />}
+            </span>
```

Comment:
```suggestion
            {validSize && <span className="help-block co-clone-pvc-modal__helpblock">
              'Size should be equal or higher than default size' 
            </span>}
```

Reply:
Did you consider using FormGroup properties for validation ?
`helperText` and `validated`